### PR TITLE
remove old style cmake

### PR DESCRIPTION
**Pull request by petricm**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -167,8 +166,7 @@ MACRO( ADD_UNIT_TEST _name )
         ADD_EXECUTABLE( test_${_name} EXCLUDE_FROM_ALL ${ARGN} )
     ENDIF()        
     TARGET_LINK_LIBRARIES( test_${_name} ${PROJECT_NAME} )
-    GET_TARGET_PROPERTY( _executable test_${_name} LOCATION )
-    ADD_TEST( t_${_name} ${_executable} )
+    ADD_TEST( t_${_name} t_${_name} )
```
Comment (andresailer) [inline CMakeLists.txt]:
```suggestion
    ADD_TEST( t_${_name} test_${_name} )
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -167,8 +166,7 @@ MACRO( ADD_UNIT_TEST _name )
         ADD_EXECUTABLE( test_${_name} EXCLUDE_FROM_ALL ${ARGN} )
     ENDIF()        
     TARGET_LINK_LIBRARIES( test_${_name} ${PROJECT_NAME} )
-    GET_TARGET_PROPERTY( _executable test_${_name} LOCATION )
-    ADD_TEST( t_${_name} ${_executable} )
+    ADD_TEST( t_${_name} test_${_name} )
```
Comment (andresailer) [inline CMakeLists.txt]:
```suggestion
    ADD_TEST( t_${_name} $<TARGET_FILE:test_${_name}> )
```

**Comment by andresailer** [comment]
@petricm  Squash and Merge ?

**Comment by petricm** [comment]
yes squash it please

**Pull request merged**